app/scripts/components/common/page-header/nav/create-dynamic-nav-menu-list.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { NavItem, NavItemType } from '../types';
+import { NavDropDownButton } from './nav-dropdown-button';
+import { NavItemExternalLink, NavItemInternalLink } from './nav-item-links';
+import { NavItemCTA } from './nav-item-cta';
+import { LinkProperties } from '$types/veda';
+
+export const createDynamicNavMenuList = (
+  navItems: NavItem[],
+  linkProperties: LinkProperties,
+  isOpen: boolean[] | undefined = undefined,
```

Comment:
```suggestion
  isOpen?: boolean[],
```
I think we should mark it as optional instead